contributing/styling.md

```diff
@@ -0,0 +1,85 @@
+[&larr; Table of Contents](../CONTRIBUTING.md)
+
+# Styling Guidelines
+🎨 How to CSS in FDS
+
+## Styling components
+
+Prefer CSS custom properties over hard coded values. The [style dictionary](https://cbinsights.github.io/form-design-system/fds-dictionary/) in fds lists available custom properties.
+
```

Comment:
Just a thought: a developer could easily interpret this as "oh this just means to make variables at the top of my file and reuse them in my declarations" rather than "try to go find existing variables for this", "if there's a close value, try to see if you can get the value changed to an existing value", etc. I can totally see people re-coding variables that already exist at the top of their specific component css file because they misunderstood what was the intention here.

Reply:
Going to make this statement explicit about preferring the vars provided by FDS